### ISSUE TITLE
Config button disabled

### ISSUE THREAD

**Issue by MaxioTech** [Jomcraft-Network/DefaultSettings#4]
Hey, I was going to use the mod in my pack, but it seems like the mods it disabled some how??
The config button is disabled and using the F7+G hotkey doesn't work either.

Screenshot: [http://prntscr.com/239mkmk](http://prntscr.com/239mkmk)

**Comment by PT400C** [comment]
Hey,

which version of the mod and Minecraft are you using? :)

PT400C

**Comment by Punsho** [comment]
I'm experiencing the same problem with 1.18.1 
Is there any other way to open the context menu? F7+G isn't working and config button is greyed out

**Comment by PT400C** [comment]
Hey @MaxioTech and @Punsho,

as it stands right now, the GUI functionality has been removed for Forge 1.17+.

Mojang changed their rendering system drastically, which forced us to settle with a command-based interface for now.

But that probably will change in the future, once we had time to adapt to the new rendering system.

PT400C

**Comment by Punsho** [comment]
How would I open that command-based interface?

**Comment by PT400C** [comment]
@Punsho It's accessible via the /ds or /defaultsettings command in a singleplayer world :)

**Comment by Punsho** [comment]
I'll see what I can do. I'd advice updating the guide to reflect this

**Comment by PT400C** [comment]
Yeah, you're absolutely right. The guides have been created when 1.16.5 was the latest version and are quite confusing nowadays. We'll update them asap

**Comment by nanonestor** [comment]
Would be nice to have some kind of guide.  Testing the current version DefaultSettings-1.18.x-2.6.1 on Forge 39.1.2 the /ds saveconfigs command does nothing of the sort.  It puts green text in chat saying that it has "saved your mod configuration settings" and the folders in /configs/defaultsettings/ contain no such config files.

**Comment by PT400C** [comment]
@nanonestor, thank you very much for that feedback! We absolutely share your opinion on that, but sadly Mojang decided to completely mess up the way we made that GUI work. If we find the time to port it to 1.17+, we'll do it ASAP.

In the meantime a proper guide is being worked on, but still WIP. So I try to help you this way: 

The /ds saveconfigs command does **NOT** do anything to your filesystem anymore! It's **NOT equal** to the export mode on 1.16! If you want to use it to ship certain config files, you have to copy them into the folder manually (for now). Execute the command **AFTER** you copied them there.

When publishing, you now have to manually remove these config files you want to ship out of the config directory of the exported filetree. If e.g. you want to ship all of them, the only directory in the configs folder should be "defaultsettings". Other than that would be some Forge files and DefaultSettings.json.


If you do not want to overwrite local changes on the user's side, /ds save -o is for you! It ships changes to new users but lets existing users keep their local changes.

Please feel free to contact me via Discord if you have further questions! PT400C#8363

PT400C

**Comment by shellduck8** [comment]
Hey, any way to change the default world generation so that the user won't have to manually click to change it? I'm making a modpack that uses the desert only biome from minecraft as world generation, but I haven't found a good way to change it.

**Comment by PT400C** [comment]
> Hey, any way to change the default world generation so that the user won't have to manually click to change it? I'm making a modpack that uses the desert only biome from minecraft as world generation, but I haven't found a good way to change it.

This feature isn't in the mod yet. I'll create a separate issue for it to be implemented.
